zwave-classifier.js

```diff
@@ -74,7 +140,34 @@ class ZWaveClassifier {
       return;
     }
 
-    let binarySensorValueId =
+    node.type = 'thing';  // Just in case it doesn't classify as anything else
+
+    const alarmValueId = node.findValueId(COMMAND_CLASS_ALARM, 1, 10);
```

Comment:
At some point (not necessarily for this PR), could we pull these magic numbers into constants, i.e. 10, 1, 3, 5, 27?